extensions/stackdriver/log/logger.cc

```diff
@@ -73,14 +73,18 @@ void setMonitoredResource(
 // Helper methods to fill destination Labels.
 void fillDestinationLabels(
```

Comment:
Done.

Reply:
Can you add more details? What labels are qualified to be put in audit log? In the future, If I am going to add a new label, I need some guidance on whether that label should be in audit log or not.

Reply:
I agree, I think it would also be good idea to add a separate function like fillAuditAndCommonDestinationLabel and fillDestinationLabels .. something on those lines..